test/services/user/forcePasswordChange.test.js

```diff
@@ -0,0 +1,90 @@
+const { expect } = require('chai');
+const app = require('../../../src/app');
```

Comment:
Codacy found an issue: [Expected 1 empty line after require statement not followed by another require.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5748939)